diff --git a/node_modules/react-native/scripts/cocoapods/new_architecture.rb b/node_modules/react-native/scripts/cocoapods/new_architecture.rb
index 3873632..f6e4cf0 100644
--- a/node_modules/react-native/scripts/cocoapods/new_architecture.rb
+++ b/node_modules/react-native/scripts/cocoapods/new_architecture.rb
@@ -171,7 +171,7 @@ class NewArchitectureHelper
         excluded_info_plist = ["/Pods", "Tests", "metainternal", ".bundle", "build/", "DerivedData/", ".xcframework", ".framework", "watchkitapp", "today-extention"]
         projectPaths.each do |projectPath|
             projectFolderPath = File.dirname(projectPath)
-            infoPlistFiles = `find #{projectFolderPath} -name "Info.plist"`
+            infoPlistFiles = `find "#{projectFolderPath}" -name "Info.plist"`
             infoPlistFiles = infoPlistFiles.split("\n").map { |f| f.strip }
 
             infoPlistFiles.each do |infoPlistFile|
